bots/tests-scan

```diff
@@ -96,6 +96,14 @@ REPO_BRANCH_CONTEXT = {
     }
 }
 
+# Random extra options
+
+extra_options_for_repo = {
+    'mvollmer/subscription-manager': {
+        'tests_invoke_pretty_logs': True
+    }
+}
```

Comment:
We probably want to name this ALL_CAPS like the other config options above?

Also, could save one jump through a hoop by just giving the additional options for tests-invoke directly?

```python
REPO_EXTRA_OPTIONS = {
    "mvollmer/subscription-manager": [ "--pretty-logs" ]
}
```

Reply:
Yes, why not.  Done.